winners-launch-content-hackathon-2023.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: 'Announcing the winners of the Xata content hackathon'
+description: 'Celebrate the standout creators and contributions from Xata's 2023 launch week content hackathon'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/content-hackathon-winners-2023.jpeg
+  alt: Xata content winners
+author: J Edwards
+date: 10-13-2023
+published: true
+slug: content-hackathon-winners-2023
+---
+
+To commemorate our most recent [launch week (Aug 28th - Sep 1st, 2023)](https://xata.io/blog/launch-week-august-2023), we called upon our vibrant community to showcase their creativity, enthusiasm, and interest for Xata. The challenge? Write a captivating blog post, create an engaging video, or simply spread the word about Xata. And you all delivered in style!
+
+The anticipation has been building, and now, we're thrilled to reveal the winners of our content hackathon.
+
+🥁 Drum roll, please! 🥁
+
+Without further ado, the winners of our hackathon (in no particular order) are:
+
+- [@Aboo_Turaab](https://twitter.com/Aboo_Turaab)
+- [@dotAadarsh](https://twitter.com/dotAadarsh)
+- [@terieyenike](https://twitter.com/terieyenike)
+- [@moe_rayo](https://twitter.com/moe_rayo)
+- [@ishnbedi](https://twitter.com/ishnbedi)
+
+Honorable mentions to:
+
+- [@rillwz](https://twitter.com/rillwz)
+- [@dev50x](https://twitter.com/dev50x)
+- [@HeroCodes](https://twitter.com/HeroCodes404)
+- [@SomanRaunak](https://twitter.com/SomanRaunak)
+
```

Comment:
I think we need one more here to complete the swag give away. 5 cash prizes, 10 swag items. We can probably use the bottom half of this list to select another honorable mention: https://xata-hq.slack.com/archives/C05PT9ZMZPZ/p1696610912289899